ingest/queries/describeDataset.rq

```diff
@@ -35,13 +40,12 @@ WHERE {
     }
   }
   OPTIONAL {
-        ?dataset dco:hasDistribution ?distribution .
-        ?distribution dco:hasFile ?file .
+        ?dataset dcat:distribution ?distribution .
         ?distribution rdfs:label ?distributionLabel .
```

Comment:
Since this is a DESCRIBE query you should be able to remove ?distributionLabel, ?distributionURL, and ?fileURL from the query.